### ISSUE TITLE
WindowedMultipole data

### ISSUE THREAD

**Issue by zhangy6x** [mit-crpg/openmc#934]
Hi, I am trying to get the doppler broadened cross-section using the on-the-fly method in your paper with `openmc.data.WindowedMultipole.from_hdf5()`. I cloned the repo and ran the scripts named `openmc-get-nndc-data` and `openmc-get-multipole-data`. But, it looks like that the multipole data files (.h5) are only available for some of the isotopes. 

I also tried it with the .h5 data files after conversion from .ace, but it didn't work. So, do you know how should I obtain the .h5 files (required by `WindowedMultipole`) of the isotopes I am interested in? Or is there any packages that could output such required files? Thanks very much!

**Comment by liangjg** [comment]
@zhangy6x thanks for your interest in windowed multipole library for Doppler broadening. As you've seen, the publicly released WMP data contains only limited nuclides. These libraries were generated using special codes from the ENDF evaluation library. The point-wise cross section cannot be directly converted into WMP format using the python API. Actually, it has not been easy to get pole parameters and generate WMP format for other nuclides because of the missing of resonance data in the evaluations. Recently, we proposed a new approach to [extend the WMP library to more nuclides](http://www.sciencedirect.com/science/article/pii/S0306454917303195). The new complete library is still under testing and is expected to be released in the future.

**Comment by zhangy6x** [comment]
@liangjg: Thanks so much for your reply and sharing the paper! Are the codes for WMP format conversion available for users?

I tried NJOY2016 to generate temperature dependent cross-sections with `openmc.data.IncidentNeutron.from_njoy`, but it took quite a while for the uranium nuclides. So for getting temperature info by resonance fitting, I found the `WindowedMultipole` class is extremely helpful! I am really looking forward to the release of a complete library.
